shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -331,6 +334,7 @@ - (void)setViewController:(FlutterViewController*)viewController {
 
 - (void)attachView {
   self.iosPlatformView->attachView();
+  [_textInputPlugin.get() setupIndirectScribbleInteraction:self.viewController];
```

Comment:
Came across this while looking into refactoring `attachView`. Does it make sense to move this line somewhere else? `setupIndirectScribbleInteraction:self` doesn't sound like should be a part of "attachView"

Reply:
Maybe there's a better place for it...iirc it's purpose here is to make sure the indirect scribble interaction is set up along with the view – that is something that needs to always be there so that we can capture scribble interactions that happen outside of text areas